mail/factories.py

```diff
@@ -0,0 +1,22 @@
+"""Factories for mail models"""
+from factory import (
+    Faker,
+    SubFactory,
+)
+from factory.django import DjangoModelFactory
+from factory.fuzzy import FuzzyText
+
+from mail.models import AutomaticEmail
+from search.factories import PercolateQueryFactory
+
+
+class AutomaticEmailFactory(DjangoModelFactory):
+    """Factory for AutomaticEmail"""
+    query = SubFactory(PercolateQueryFactory)
+    enabled = Faker('boolean')
+    email_subject = FuzzyText()
+    email_body = FuzzyText()
+    sender_name = FuzzyText()
```

Comment:
There's some faker class/method for generating names. If it's easy enough to find that and implement here, go for it. Not very important, obviously